client/src/components/layout/Header.tsx

```diff
@@ -90,6 +105,20 @@ const Header: FC<HeaderType> = props => {
               onClick={() => handleBack(navInfo.backPath)}
             />
           )}
+          {navInfo.showDatabaseSelector ? (
+            <CustomSelector
+              label="Database"
+              value={database}
+              onChange={async (e: { target: { value: unknown } }) => {
+                const database = e.target.value as string;
+                await useDatabase(database);
+                setDatabase(database);
+              }}
```

Comment:
Do we need to redirect to the home page like dashboard when database change?

Prevent us from having some data problems on some pages.